earthly/rust/config.toml

```diff
@@ -13,14 +13,14 @@ rustflags = [
 linker = "clang"
 rustflags = [
     "-C", "link-arg=-fuse-ld=/usr/bin/mold",
-    # "-C", "target-feature=+crt-static" - Doesn't work for nextest.
+    # "-C", "target-feature=+crt-static" - proc-macro doesn't work with it. `https://github.com/rust-lang/rust/issues/78210`
 ]
 
 [target.aarch64-unknown-linux-gnu]
 linker = "clang"
 rustflags = [
     "-C", "link-arg=-fuse-ld=/usr/bin/mold",
-    "-C", "target-feature=+crt-static"
+    # "-C", "target-feature=+crt-static" - proc-macro doesn't work with it. `https://github.com/rust-lang/rust/issues/78210`
```

Comment:
Thats not unexpected.  The key point is to make sure when your target is built its NOT built with this target.  This target can ONLY be used to build the compiler and its tooling.

Reply:
But it does not work for nextest, the same issue as with `x86_64-unknown-linux-gnu`

Reply:
nextest is just a tool, its ok to build it with the GNU tools.  I hit this with x86 build as well.  This is OK to disable for getting nextest to run.